syntax = "proto3";
service AiService {
    rpc PredictComment (PredictCommentRequest) returns (PredictCommentReply) {}
  }


  message PredictCommentRequest {
    string data = 2;
  }


  message PredictCommentReply {
    string data = 1;
  }
